lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -422,8 +422,8 @@ stash all the time.
 
 =head2 Restrictive placeholders
 
-The easiest way to make placeholders more restrictive are alternatives, you
-just make a list of possible values.
+The easiest way to make placeholders more restrictive is to use
+alternatives: you just make a list of possible values.
```

Comment:
this would be a semi-colon I believe, as many of these suggestions could also be